conda-forge.yml

```diff
@@ -1,7 +1,11 @@
+build_platform:
```

Comment:
I'm not sure if cross-compiling actually works but I'm happy to merge and see if anyone complains.

Reply:
@chrisburr, at least on aarch64/ppc64le, the tests are run in emulation
https://github.com/conda-forge/cling-feedstock/blob/73493a1c26a7c673e70057ae6b1a80e99223d790/conda-forge.yml#L11
and that includes execution tests
https://github.com/conda-forge/cling-feedstock/blob/73493a1c26a7c673e70057ae6b1a80e99223d790/recipe/meta.yaml#L58-L59
🙃